app/Actions/Photo/Pipes/Shared/NotifyAlbums.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Actions\Photo\Pipes\Shared;
+
+use App\Actions\User\Notify;
+use App\Contracts\PhotoCreate\SharedPipe;
+use App\DTO\PhotoCreate\DuplicateDTO;
+
+/**
+ * Notify by email if a picture has been added.
+ */
+class NotifyAlbums implements SharedPipe
+{
+	public function handle(DuplicateDTO $state, \Closure $next): DuplicateDTO
```

Comment:
I guess these can't use a more generic model? (allowing everything to call the same code)

Reply:
It will be more generic later, I will use Union type in for SharedPipe later, but this is not in the scope of this PR. :)